src/App/component/AddProjectDialog.jsx

```diff
@@ -159,4 +83,4 @@ export default function AddProjectDialog({open, reloadProjects, handleClose}) {
       </DialogActions>
     </Dialog>
   )
-}
+}
```

Comment:
Suggest add a new line in the end of this file.
```suggestion
}

```